src/devtools/pachdev/main.go

```diff
@@ -0,0 +1,138 @@
+package main
```

Comment:
I'm open to either option, but you might like making this a starpach subcommand.   One binary to rule them all.  Oh and there's also pachctl ;)